adafruit_led_animation/animation/comet.py

```diff
@@ -68,12 +68,16 @@ def __init__(
         pixel_object,
         speed,
         color,
-        tail_length=10,
+        tail_length=0,
         reverse=False,
         bounce=False,
         name=None,
     ):
-        self._tail_length = tail_length + 1
+        if tail_length is 0:
```

Comment:
`tail_length == 0` instead of `is` should fix the pylint complaint.

Reply:
I knew this, that's what I had first, but PyCharm was complaining about a thing, so I went against what I knew was correct. Fixed it as you were posting this.